### ISSUE TITLE
Write about new topics in ArXiV on 2022-03-01

### ISSUE THREAD

**Issue by wanted2** [wanted2/caineng.in#465]
**Is your feature request related to a problem? Please describe.**
*Write about new topics in ArXiV on 2022-03-01*
**Describe the solution you'd like**
* Topic 0: 
  * _Patterns of near-crash events in a naturalistic driving dataset:  applying rules mining_
  * _Iterated Belief Change, Computationally_
  * _Parameterized Intractability for Multi-Winner Election under the  Chamberlin-Courant Rule and the Monroe Rule_
  * _Automated Reasoning in Non-classical Logics in the TPTP World_
  * _Formulating Beurling LASSO for Source Separation via Proximal Gradient  Iteration_
  * _Rethinking Immersive Virtual Reality and Empathy_
  * _UnifiedQA-v2: Stronger Generalization via Broader Cross-Format Training_
  * _A General Framework for Modelling Conditional Reasoning -- Preliminary  Report_
  * _On change of measure inequalities for $f$-divergences_
  * _Program Synthesis for the OEIS_
* Topic 1: 
  * _Reproducibility in Optimization: Theoretical Framework and Limits_
  * _Long-Time Convergence and Propagation of Chaos for Nonlinear MCMC_
  * _Minimax in Geodesic Metric Spaces: Sion's Theorem and Algorithms_
  * _Differentially Private Regression with Unbounded Covariates_
  * _Single Trajectory Nonparametric Learning of Nonlinear Dynamics_
  * _A PDE-Based Analysis of the Symmetric Two-Armed Bernoulli Bandit_
  * _Generalized Bayesian Additive Regression Trees Models: Beyond  Conditional Conjugacy_
  * _Reachability analysis in stochastic directed graphs by reinforcement  learning_
  * _Almost Optimal Proper Learning and Testing Polynomials_
  * _Grassmann Stein Variational Gradient Descent_
* Topic 2: 
  * _Confidence Calibration for Object Detection and Segmentation_
  * _BeautifAI -- A Personalised Occasion-oriented Makeup Recommendation  System_
  * _On the Overlooked Significance of Underutilized Contextual Features in  Recent News Recommendation Models_
  * _Unintended Bias in Language Model-driven Conversational Recommendation_
  * _Movie Recommender System using critic consensus_
  * _Towards Fair Recommendation in Two-Sided Platforms_
  * _CSSR: A Context-Aware Sequential Software Service Recommendation Model_
  * _Sequential Item Recommendation in the MOBA Game Dota 2_
  * _State of the Art of User Simulation approaches for conversational  information retrieval_
* Topic 3: 
  * _Graph Representation Learning for Spatial Image Steganalysis_
  * _Improving Content-Aware Video Streaming in Congested Networks with  In-Network Computing_
  * _Punctuation Restoration_
  * _Banding vs. Quality: Perceptual Impact and Objective Assessment_
  * _GenéLive! Generating Rhythm Actions in Love Live!_
  * _MMSys'22 Grand Challenge on AI-based Video Production for Soccer_
  * _Towards Using Clothes Style Transfer for Scenario-aware Person Video  Generation_
  * _Promoting Mental Well-Being for Audiences in a Live-Streaming Game by  Highlight-Based Bullet Comments_
  * _How Camera Placement Affects Gameplay in Video Games_
* Topic 4: 
  * _PARM: A Paragraph Aggregation Retrieval Model for Dense  Document-to-Document Retrieval_
  * _Case law retrieval: problems, methods, challenges and evaluations in the  last 20 years_
  * _Transfer Learning Approaches for Building Cross-Language Dense Retrieval  Models_
  * _Improving BERT-based Query-by-Document Retrieval with Multi-Task  Optimization_
  * _Literature Review of the Pioneering Approaches in Cloud-based Search  Engines Powered by LETOR Techniques_
  * _Mirror Matching: Document Matching Approach in Seed-driven Document  Ranking for Medical Systematic Reviews_
  * _Audio Retrieval with Natural Language Queries: A Benchmark Study_
  * _Targeted Trojan-Horse Attacks on Language-based Image Retrieval_
  * _Establishing Strong Baselines for TripClick Health Retrieval_
  * _Grep-BiasIR: A Dataset for Investigating Gender Representation-Bias in  Information Retrieval Results_
* Topic 5: 
  * _Emotion Recognition from Multiple Modalities: Fundamentals and  Methodologies_
  * _MELONS: generating melody with long-term structure using transformers  and structure graph_
  * _Dual Learning Music Composition and Dance Choreography_
  * _Symbolic Music Loop Generation with VQ-VAE_
  * _Music Sentiment Transfer_
  * _Theme Transformer: Symbolic Music Generation with Theme-Conditioned  Transformer_
  * _Predicting emotion from music videos: exploring the relative  contribution of visual and auditory information to affective responses_
  * _Learning long-term music representations via hierarchical contextual  constraints_
  * _A Novel Multi-Task Learning Method for Symbolic Music Emotion  Recognition_
* Topic 6: 
  * _Trustworthy Anomaly Detection: A Survey_
  * _Multimedia Datasets for Anomaly Detection: A Survey_
  * _Boosted Dense Retriever_
  * _Generating Clarifying Questions for Query Refinement in Source Code  Search_
  * _Query Suggestion for Click-Absent Queries in Enterprise Search_
  * _Event-centric Query Suggestion for Online News_
* Topic 7: 
  * _Point cloud completion on structured feature map with feedback network_
  * _Diversity in the Music Listening Experience: Insights from Focus Group  Interviews_
  * _PCSCNet: Fast 3D Semantic Segmentation of LiDAR Point Cloud for  Autonomous Car using Point Convolution and Sparse Convolution Network_
  * _Snowflake Point Deconvolution for Point Cloud Completion and Generation  with Skip-Transformer_
  * _Patch-Based Deep Autoencoder for Point Cloud Geometry Compression_
  * _MusIAC: An extensible generative framework for Music Infilling  Applications with multi-level Control_
  * _Point Cloud Denoising via Momentum Ascent in Gradient Fields_
  * _ChMusic: A Traditional Chinese Music Dataset for Evaluation of  Instrument Recognition_
  * _Malakai: Music That Adapts to the Shape of Emotions_
  * _Video as Conditional Graph Hierarchy for Multi-Granular Question  Answering_
* Topic 8: 
  * _Improving Robustness of Convolutional Neural Networks Using Element-Wise  Activation Scaling_
  * _Privacy Leakage of Adversarial Training Models in Federated Learning  Systems_
  * _Semi-Implicit Hybrid Gradient Methods with Application to Adversarial  Robustness_
  * _Generative Adversarial Network-Driven Detection of Adversarial Tasks in  Mobile Crowdsensing_
  * _Real-time Over-the-air Adversarial Perturbations for Digital  Communications using Deep Neural Networks_
  * _Mitigating Closed-model Adversarial Examples with Bayesian Neural  Modeling for Enhanced End-to-End Speech Recognition_
  * _ARIA: Adversarially Robust Image Attribution for Content Provenance_
  * _Towards Robust Neural Image Compression: Adversarial Attack and Model  Finetuning_
* Topic 9: 
  * _Causal discovery for observational sciences using supervised machine  learning_
  * _Deep End-to-end Causal Inference_
  * _Causal Analysis Framework for Recommendation_
  * _Learning Infomax and Domain-Independent Representations for Causal  Effect Inference with Real-World Data_
  * _Interpolation and Regularization for Causal Learning_
  * _Stochastic Causal Programming for Bounding Treatment Effects_
  * _Joint Geometry and Color Projection-based Point Cloud Quality Metric_
  * _A Semi-Synthetic Dataset Generation Framework for Causal Inference in  Recommender Systems_
  * _Effect Identification in Cluster Causal Diagrams_
  * _Learning Latent Causal Dynamics_
* Topic 10: 
  * _Domain-Augmented Domain Adaptation_
  * _Multi-Source Unsupervised Domain Adaptation via Pseudo Target Domain_
  * _Transferrable Contrastive Learning for Visual Domain Adaptation_
  * _Conditional Extreme Value Theory for Open Set Video Domain Adaptation_
  * _GPL: Generative Pseudo Labeling for Unsupervised Domain Adaptation of  Dense Retrieval_
  * _Graph-Relational Domain Adaptation_
  * _Cost-effective Framework for Gradual Domain Adaptation with  Multifidelity_
  * _BP-Triplet Net for Unsupervised Domain Adaptation: A Bayesian  Perspective_
  * _Domain Randomization for Object Counting_
* Topic 11: 
  * _Musical Audio Similarity with Self-supervised Convolutional Neural  Networks_
  * _Multimodal Incremental Transformer with Visual Grounding for Visual  Dialogue Generation_
  * _R-Theta Local Neighborhood Pattern for Unconstrained Facial Image  Recognition and Retrieval_
  * _Guide Local Feature Matching by Overlap Estimation_
  * _Local Sliced-Wasserstein Feature Sets for Illumination-invariant Face  Recognition_
  * _Local Directional Gradient Pattern: A Local Descriptor for Face  Recognition_
  * _VLAD-VSA: Cross-Domain Face Presentation Attack Detection with  Vocabulary Separation and Adaptation_
  * _Local Quadruple Pattern: A Novel Descriptor for Facial Image Recognition  and Retrieval_
* Topic 12: 
  * _PMP-Net++: Point Cloud Completion by Transformer-Enhanced Multi-step  Point Moving Paths_
  * _Compressed Geometric Arrays for Point Cloud Processing_
  * _3D Point Cloud Reconstruction and SLAM as an Input_
  * _Point Cloud Generation with Continuous Conditioning_
  * _The Winning Solution to the iFLYTEK Challenge 2021 Cultivated Land  Extraction from High-Resolution Remote Sensing Image_
  * _A Smoothing and Thresholding Image Segmentation Framework with Weighted  Anisotropic-Isotropic Total Variation_
* Topic 13: 
  * _Distantly Supervised Semantic Text Detection and Recognition for  Broadcast Sports Videos Understanding_
  * _Anomalib: A Deep Learning Library for Anomaly Detection_
  * _Lightweight Multi-Drone Detection and 3D-Localization via YOLO_
  * _Synthesizing Photorealistic Images with Deep Generative Learning_
  * _Transformers in Medical Image Analysis: A Review_
  * _CAISE: Conversational Agent for Image Search and Editing_
* Topic 14: 
  * _Biased or Not?: The Story of Two Search Engines_
  * _Tree-based Search Graph for Approximate Nearest Neighbor Search_
  * _Full-privacy secured search engine empowered by efficient genome-mapping  algorithms_
  * _DL-SLOT: Dynamic Lidar SLAM and Object Tracking Based On Graph  Optimization_
  * _Amodal Panoptic Segmentation_
  * _Perceptual Consistency in Video Segmentation_
* Topic 15: 
  * _Boosting Barely Robust Learners: A New Perspective on Adversarial  Robustness_
  * _Multi-Cue Adaptive Emotion Recognition Network_
  * _A Prospective Approach for Human-to-Human Interaction Recognition from  Wi-Fi Channel Data using Attention Bidirectional Gated Recurrent Neural  Network with GUI Application Implementation_
* Topic 16: 
  * _Optimal Clustering with Bandit Feedback_
  * _Use Image Clustering to Facilitate Technology Assisted Review_
* Topic 17: 
  * _Unsupervised Graph Poisoning Attack via Contrastive Loss  Back-propagation_
  * _AF$_2$: Adaptive Focus Framework for Aerial Imagery Segmentation_
* Topic 18: 
  * _Clustering by Hill-Climbing: Consistency Results_
  * _Using the left Gram matrix to cluster high dimensional data_
  * _A sampling-based approach for efficient clustering in large datasets_
  * _Web-Based File Clustering and Indexing for Mindoro State University_
  * _Mind Your Clever Neighbours: Unsupervised Person Re-identification via  Adaptive Clustering Relationship Modeling_
  * _Clustering by the Probability Distributions from Extreme Value Theory_
  * _Human Motion Detection Using Sharpened Dimensionality Reduction and  Clustering_
  * _Self-Evolutionary Clustering_
* Topic 19: 
  * _A Robust Multi-Objective Bayesian Optimization Framework Considering  Input Uncertainty_
  * _Dissecting the impact of different loss functions with gradient surgery_
  * _On the Effectiveness of Sampled Softmax Loss for Item Recommendation_
  * _Cyclical Focal Loss_
  * _General Cyclical Training of Neural Networks_
* Topic 20: 
  * _Provable Stochastic Optimization for Global Contrastive Learning: Small  Batch Does Not Harm Performance_
  * _STEREO: Scientific Text Reuse in Open Access Publications_
* Topic 21: 
* Topic 22: 
  * _Unsupervised physics-informed disentanglement of multimodal data for  high-throughput scientific discovery_
  * _Language technology practitioners as language managers: arbitrating data  bias and predictive bias in ASR_
* Topic 23: 
  * _Differentially Private Estimation of Heterogeneous Causal Effects_
* Topic 24: 
  * _VCVTS: Multi-speaker Video-to-Speech synthesis via cross-modal knowledge  transfer from voice conversion_
  * _A Survey on Patients Privacy Protection with Stganography and Visual  Encryption_
* Topic 25: 
  * _repro_eval: A Python Interface to Reproducibility Measures of  System-oriented IR Experiments_
* Topic 26: 
  * _Speaker Adaptation Using Spectro-Temporal Deep Features for Dysarthric  and Elderly Speech Recognition_
  * _Speaker Identity Preservation in Dysarthric Speech Reconstruction by  Adversarial Speaker Adaptation_
  * _E-ffective: A Visual Analytic System for Exploring the Emotion and  Effectiveness of Inspirational Speeches_
  * _AISHELL-NER: Named Entity Recognition from Chinese Speech_
  * _Similarity search on neighbor's graphs with automatic Pareto optimal  performance and minimum expected quality setups based on hyperparameter  optimization_
  * _Implementing simple spectral denoising for environmental audio  recordings_
* Topic 27: 
  * _Efficient and Differentiable Conformal Prediction with General Function  Classes_
  * _LSTM-RPA: A Simple but Effective Long Sequence Prediction Algorithm for  Music Popularity Prediction_
  * _Selective Credit Assignment_
  * _Adiabatic Quantum Computing for Multi Object Tracking_
* Topic 28: 
  * _Temporal Subtyping of Alzheimer's Disease Using Medical Conditions  Preceding Alzheimer's Disease Onset in Electronic Health Records_
* Topic 29: 
* Topic 30: 
* Topic 31: 
* Topic 32: 
  * _Visual Acoustic Matching_
  * _Bidding Agent Design in the LinkedIn Ad Marketplace_
* Topic 33: 
  * _Guidelines and evaluation for clinical explainable AI on medical image  analysis_
  * _Deconstructing Distributions: A Pointwise Framework of Learning_
* Topic 34: 
  * _SLIDE: a surrogate fairness constraint to ensure fairness consistency_
* Topic 35: 
  * _MultiRes-NetVLAD: Augmenting Place Recognition Training with  Low-Resolution Imagery_
  * _Detecting Harmful Memes and Their Targets_
* Topic 36: 
  * _Differential Privacy and Fairness in Decisions and Learning Tasks: A  Survey_
* Topic 37: 
  * _Cross-Task Knowledge Distillation in Multi-Task Recommendation_
* Topic 38: 
* Topic 39: 
* Topic 40: 
  * _Geometry of the Minimum Volume Confidence Sets_
  * _Reasoning with fuzzy and uncertain evidence using epistemic random fuzzy  sets: general framework and practical models_
  * _Transformers in Time Series: A Survey_
* Topic 41: 
  * _Learning to Identify Perceptual Bugs in 3D Video Games_
  * _How player and opponent personalities influence cooperative gameplay_
  * _Fighting Game Commentator with Pitch and Loudness Adjustment Utilizing  Highlight Cues_
  * _The four-fifths rule is not disparate impact: a woeful tale of epistemic  trespassing in algorithmic fairness_
* Topic 42: 
  * _Integrating psychotherapy practices and gamified elements in novel game  mechanics for stress relief_
* Topic 43: 
  * _Extraction of Product Specifications from the Web -- Going Beyond Tables  and Lists_
  * _MAVE: A Product Dataset for Multi-source Attribute Value Extraction_
  * _Can Online Customer Reviews Help Design More Sustainable Products? A  Preliminary Study on Amazon Climate Pledge Friendly Products_
  * _Decision support system for distributed manufacturing based on  input-output analysis and economic complexity_
  * _SAGE: SLAM with Appearance and Geometry Prior for Endoscopy_
* Topic 44: 
  * _Finding Safe Zones of policies Markov Decision Processes_
  * _A hybrid 2-stage vision transformer for AI-assisted 5 class pathologic  diagnosis of gastric endoscopic biopsies_
* Topic 45: 
  * _Wisdom of (Binned) Crowds: A Bayesian Stratification Paradigm for Crowd  Counting_
* Topic 46: 
  * _On PAC-Bayesian reconstruction guarantees for VAEs_
* Topic 47: 
  * _Multiscale Crowd Counting and Localization By Multitask Point  Supervision_
* Topic 48: 
  * _Sobolev Transport: A Scalable Metric for Probability Measures with Graph  Metrics_
* Topic 49: 

**Describe alternatives you've considered**
None
**Additional context**
None

**Comment by wanted2** [comment]
This issue was created by an aggregate batch
